.github/workflows/hybrid-integration-test.yaml

```diff
@@ -0,0 +1,68 @@
+name: integration test
+
+on:
+  push:
```

Comment:
mchurichi said:

please add the paths, so this is run only when pushing hybrid attestor code
  push:
    paths:
    - 'hybrid/**'